ui/client/src/contexts/ApplicationContext.tsx

```diff
@@ -32,7 +32,7 @@ export const ApplicationContextProvider = ({ children, colorMode }: Props) => {
         return 0;
       }),
     refetchInterval: constants.UPDATE_FREQUENCY_MILLISECONDS,
-    retry: false
+    retry: true
```

Comment:
One issue that I see with turning `retry` to `true` is that it seems that in that case `error` will always be null. This means that if the UI cannot reach the Paladin node (for whatever reason) instead of showing an error message, the UI will just freeze. 
@dechdev do you happen to know what would be a good way to handle this scenario?

Reply:
Setting `retry: true` will retry the query indefinitely ad. I don't think we want this. Instead, we should have a global max number of retries, and use the retry callback function, seen [here](https://tanstack.com/query/v4/docs/framework/react/reference/useQuery). 

```
retry: boolean | number | (failureCount: number, error: TError) => boolean
- If false, failed queries will not retry by default.
- If true, failed queries will retry infinitely.
- If set to a number, e.g. 3, failed queries will retry until the failed query count meets that number.
defaults to 3 on the client and 0 on the server
```